tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -128,6 +127,16 @@ if [ "$pa_color" ] && [ "$pa_size" ]; then
 	"
 fi
 
+if [ "$pa_color" ]; then
```

Comment:
I'm uncertain if I follow you, @gigitux. The *.txt files inside the bin folder, are the individual blocks, e.g. Active Filters block, All Products blocks, etc. Shouldn't the test products be decoupled from the individual blocks?

Reply:
Sorry if I wasn't clear. As you can see, now we have a specific command for each `.txt` file that is in the `bin` folder. https://github.com/woocommerce/woocommerce-blocks/blob/90760e4be19c5e719d8ae20fcb372d01ce63c191/tests/e2e-pw/bin/test-env-setup.sh/#L113-L153

It is easy to maintain because we have few `.txt` files, but it will not be when there are more. So, I'm suggesting creating a dedicated script that automatically inserts all the posts/pages reading all the `.txt`.

Reply:
Just to clarify, it isn't something that we should do in this PR, but after reading the code it came to my mind!

Reply:
Thanks for clarifying this! I thought you were referring to `if [ "$pa_color" ]; then`, as I see that part highlighted. Automating the loading of the *.txt files definitely makes sense. The current approach looks very redundant.

Reply:
@nielslange, thanks for the feedback! Issue created! https://github.com/woocommerce/woocommerce-blocks/issues/9646